source/includes/indexes/single-field.scala

```diff
@@ -0,0 +1,39 @@
+// start-single-index-imports
+import org.mongodb.scala._
+import org.mongodb.scala.model.Indexes
+import org.mongodb.scala.model.IndexOptions._
+import org.mongodb.scala.model.Projections._
+import org.mongodb.scala.model.Sorts._
+import org.mongodb.scala.model.Filters._
+// end-single-index-imports
+
+
+object SingleFieldIndex {
+
+    def main(args: Array[String]): Unit = {
+
+    // Replace the placeholder with your Atlas connection string
+    val connectionString = "<connection string>";
+
+    // Create a new client and connect to the server
+    val mongoClient = MongoClient(connectionString)
+    val database = mongoClient.getDatabase("sample_training")
+    val collection = database.getCollection("companies")
+
+    // start-index-single
+    collection.createIndex(Indexes.ascending("title")).printResults()
```

Comment:
Is `printResults()` required here? If not I think we can omit it from the example since we don't show any output (or if so maybe we can show the output)

Reply:
I need to ask the tech reviewer about this. It doesn't seem to work without appending print results for some reason.